testpilot/frontend/static-src/app/views/experiment-page.js

```diff
@@ -273,7 +267,10 @@ export default PageView.extend({
       eventLabel: 'Disable Experiment'
     });
 
-    const queryParams = querystring.stringify({ref: 'disable', experiment: this.model.title});
+    const installed = Object.keys(app.me.installed);
```

Comment:
@lmorchard yeah, I thought so too. I think that https://github.com/mozilla/testpilot/blob/master/testpilot/frontend/static-src/app/models/me.js#L16 might still be forcing it to into an object.